### DISCUSSION TITLE
Python equivalent of '--debug-PEC'

### DISCUSSION THREAD

**Discussion by ajmeratwal** [thliebig/openEMS-Project#211] [Q&A Usage] [answered]
Hello.  Is there a way of invoking the "--debug-PEC" option through the Python interface ?  I can see how it is done through Matlab, or the command-line, but not sure how to invoke the option via Python.  Thanks in advance.

**Comment by LubomirJagos42** [comment, marked as answer]
Hi, there is debug_pec parameter in openEMS.Run(...), here:
```python
FDTD.Run(Sim_Path, verbose=3, cleanup=True, setup_only=setup_only, debug_pec='''True/False''')
```
